src/wallet/api/walletServiceAxios.ts

```diff
@@ -20,11 +25,11 @@ import HathorWalletServiceWallet from '../wallet';
  *
  * @param {number} timeout Timeout in milliseconds for the request
  */
-export const axiosInstance = async (wallet: HathorWalletServiceWallet | null = null, timeout: number = TIMEOUT) => {
+export const axiosInstance = async (wallet: HathorWalletServiceWallet | null = null, network: Network, timeout: number = TIMEOUT) => {
```

Comment:
Should we refactor the parameters and set `network` as the first and only required parameter and add `wallet` and `timeout` as an optional options parameter just like we've been doing with optional parameters in the lib?

Reply:
I agree, done in 2b50b0be879e9da40172f4463b9e63dcd783a899